packages/custom-webpack/src/common.ts

```diff
@@ -23,6 +23,14 @@ export const indexHtmlTransformFactory: (
   context: BuilderContext
 ) => IndexHtmlTransform = ({ indexTransform }, { workspaceRoot, target }) => {
   if (!indexTransform) return null;
+  if (indexTransform.endsWith('.ts')) {
```

Comment:
Should be externalized to a separate function (in this file) and reused [here](https://github.com/just-jeb/angular-builders/blob/master/packages/custom-webpack/src/custom-webpack-builder.ts#L55-L62)

Reply:
I created a function in `custom-webpack-builder` and reused here to avoid an overlap importing. If not OK, I think it better to write a new file 🤔